### ISSUE TITLE
Autocollapse onscroll

### ISSUE THREAD

**Issue by 97balakrishnan** [shrikanth7698/Collapsible-Calendar-View-Android#5]
Enhancement can be made with autocollapse on scroll
I don't think its already there.

**Comment by nishchalraj** [comment]
Yup, the autocollapse can be introduced using a method for that. You can fork and write for that if you want to. Then ask for pull request. :)
